From bd7a5c5dc082707dd62083514bbb88c6f78e5bdb Mon Sep 17 00:00:00 2001
From: John David Anglin <danglin@gcc.gnu.org>
Date: Fri, 20 Sep 2019 21:47:56 +0000
Subject: [PATCH] pa.c (pa_trampoline_init): Remove spurious extended
 character.

	* config/pa/pa.c (pa_trampoline_init): Remove spurious extended
	character.

From-SVN: r276007
---
 gcc/config/pa/pa.c | 2 +-

--- a/gcc/config/pa/pa.c
+++ b/gcc/config/pa/pa.c
@@ -10152,7 +10152,7 @@ pa_trampoline_init (rtx m_tramp, tree fndecl, rtx chain_value)
     }
 
 #ifdef HAVE_ENABLE_EXECUTE_STACK
-  emit_library_call (gen_rtx_SYMBOL_REF (Pmode, "__enable_execute_stack"),
-		     LCT_NORMAL, VOIDmode, 1, XEXP (m_tramp, 0), Pmode);
+  emit_library_call (gen_rtx_SYMBOL_REF (Pmode, "__enable_execute_stack"),
+		      LCT_NORMAL, VOIDmode, 1, XEXP (m_tramp, 0), Pmode);
 #endif
 }
-- 
2.25.0

